src/cpu/x64/jit_generator.hpp

```diff
@@ -761,6 +771,16 @@ class jit_generator : public Xbyak::CodeGenerator, public c_compatible {
         vsubps(x, op1, op2);
     }
 
+    void uni_vaddsubps(const Xbyak::Xmm &x, const Xbyak::Operand &op1,
+            const Xbyak::Operand &op2) {
+        if (is_valid_isa(avx)) {
+            vaddsubps(x, op1, op2);
+        } else {
+            if (!x.isEqualIfNotInherited(op1)) { movups(x, op1); }
```

Comment:
Also would be nice to have movups(x, op1); on next line along with braces, because it makes it harder to read

Reply:
Done

Reply:
Issue is resolved. Cannot resolve comments in this repository yet